charts/sentry/templates/snuba/deployment-snuba-profiling-profiles-consumer.yaml

```diff
@@ -160,8 +160,8 @@ spec:
         securityContext:
 {{ toYaml .Values.snuba.profilingProfilesConsumer.containerSecurityContext | indent 12 }}
 {{- end }}
-{{- if .Values.profilingFunctionsConsumer.sidecars }}
-{{ toYaml .Values.profilingFunctionsConsumer.sidecars | indent 6 }}
+{{- if .Values.snuba.profilingFunctionsConsumer.sidecars }}
```

Comment:
@ServerSideHannes please correct `profilingFunctionsConsumer` to `profilingProfilesConsumer` for this file

Reply:
And please add default commented values `# sidecars: []` for profilingFunctionsConsumer and profilingFunctionsConsumer to `values.yaml`